x/compute/internal/keeper/keeper_test.go

```diff
@@ -840,14 +841,6 @@ func TestExecuteWithStorageLoop(t *testing.T) {
 	ctx = ctx.WithGasMeter(sdk.NewGasMeter(gasLimit))
 	require.Equal(t, uint64(0), ctx.GasMeter().GasConsumed())
 
-	// ensure we get an out of gas panic
-	defer func() {
-		r := recover()
-		require.NotNil(t, r)
-		_, ok := r.(sdk.ErrorOutOfGas)
-		require.True(t, ok, "%+v", r)
-	}()
```

Comment:
Why delete this?

Reply:
Because the execution now returns a wasmerror gracefully rather than returning a panic

Reply:
I think the cosmos-sdk panics when out of gas, so when the execution returns the gas amount from WASM and we do `consumeGas(ctx, gasUsed)` it panics there... Am I wrong?

Reply:
This is because when calling `use_gas_externally` it doesn't actually consume gas on the sdk side (that would require another ocall), but just bumps up the gas tracker in the enclave. In this test now that gets hit, not the sdk gas, so the error is actually a graceful out of gas, rather than the panic on the sdk side

Reply:
I see, thanks for explaining